README.md

```diff
@@ -96,6 +96,8 @@ The full JDBC url should look like this:
 jdbc:mysql://google/<DATABASE_NAME>?cloudSqlInstance=<INSTANCE_CONNECTION_NAME>&socketFactory=com.google.cloud.sql.mysql.SocketFactory&useSSL=false&user=<MYSQL_USER_NAME>&password=<MYSQL_USER_PASSWORD>
```

Comment:
can we remove the `google` host here too?

Reply:
Done.